web/docs/setup/deploy/README.md

```diff
@@ -6,3 +6,6 @@ it up in a runtime.
 We currently support the following runtimes:
 1. [AWS](deploy/aws)
 2. [Docker](deploy/docker)
+
+We also provide an Ansible role for bare-metal and traditional virtual machines
+in the [ansible directory](/ansible).
```

Comment:
This link does not work when you render it locally, as it points to the endpoint `/ansible`, which is not a page we have in our docs.